src/ts/uiconfig.ts

```diff
@@ -96,4 +96,8 @@ export interface UIConfig {
    * Default: false
    */
   enterFullscreenOnInitialPlayback?: boolean;
+  /**
+  * Forces subtitle-labels back into their respective container if they overflow and are therefore cropped.
+  */
+  forceSubtitlesIntoViewContainer?: boolean;
```

Comment:
There seems to be an indentation problem:

```suggestion
  /**
   * Forces subtitle-labels back into their respective container if they overflow and are therefore cropped.
   */
  forceSubtitlesIntoViewContainer?: boolean;
```